### PR TITLE
Additional reagents associated with doi: 10.1101/2024.02.28.582563.

### PR DESCRIPTION
**Pull request by zivy**
Adding additional reagent validations for new species: Turquoise Killifish, African Clawed Frog (Xenopus laevis).

Reagents used in paper: A. Kothurkar et al., "Iterative Bleaching Extends Multiplexity (IBEX) imaging facilitates simultaneous identification of all cell types in the vertebrate retina", bioRxiv, 2024.

### CODE REVIEW

File: docs/supporting_material/Calretinin_Unconjugated/0009-0000-2047-4228.md
Comment (radtkea) [inline docs/supporting_material/Calretinin_Unconjugated/0009-0000-2047-4228.md:1]: Remove note from Turquoise killifish. This applies to zebrafish entries only. Tissue state should be changed to NA.

File: docs/supporting_material/Glutamine_synthetase_CoraLite_Plus_AF488/0009-0000-2047-4228.md
Comment (radtkea) [inline docs/supporting_material/Glutamine_synthetase_CoraLite_Plus_AF488/0009-0000-2047-4228.md:1]: Remove note from Turquoise killifish and African Clawed Frog. This applies to zebrafish entries only. Tissue state should be changed to NA for both entries.

File: docs/supporting_material/HuC_D_Unconjugated/0009-0000-2047-4228.md
Comment (radtkea) [inline docs/supporting_material/HuC_D_Unconjugated/0009-0000-2047-4228.md:1]: Remove note from Turquoise killifish and African Clawed Frog. This applies to zebrafish entries only. Tissue state should be changed to NA for both entries.

File: docs/supporting_material/PAX6_Unconjugated/0009-0000-2047-4228.md
Comment (radtkea) [inline docs/supporting_material/PAX6_Unconjugated/0009-0000-2047-4228.md:1]: Remove note from Turquoise killifish and African Clawed Frog. This applies to zebrafish entries only. Tissue state should be changed to NA for both entries.

File: docs/supporting_material/PCNA_Unconjugated/0009-0000-2047-4228.md
Comment (radtkea) [inline docs/supporting_material/PCNA_Unconjugated/0009-0000-2047-4228.md:1]: Remove note from Turquoise killifish and African Clawed Frog. This applies to zebrafish entries only. Tissue state should be changed to NA for both entries.

File: docs/supporting_material/Peripherin_2_Unconjugated/0009-0000-2047-4228.md
Comment (radtkea) [inline docs/supporting_material/Peripherin_2_Unconjugated/0009-0000-2047-4228.md:1]: Remove note from Turquoise killifish and African Clawed Frog. This applies to zebrafish entries only. Tissue state should be changed to NA for both entries.

File: docs/supporting_material/Zpr-1_Unconjugated/0009-0000-2047-4228.md
Comment (radtkea) [inline docs/supporting_material/Zpr-1_Unconjugated/0009-0000-2047-4228.md:1]: Remove note from Turquoise killifish. This applies to zebrafish entries only. Tissue state should be changed to NA.

File: docs/supporting_material/Zpr-3_Unconjugated/0009-0000-2047-4228.md
Comment (radtkea) [inline docs/supporting_material/Zpr-3_Unconjugated/0009-0000-2047-4228.md:1]: Remove note from African Clawed Frog. This applies to zebrafish entries only. Tissue state should be changed to NA.

File: docs/supporting_material/Zrf-1_Unconjugated/0009-0000-2047-4228.md
Comment (radtkea) [inline docs/supporting_material/Zrf-1_Unconjugated/0009-0000-2047-4228.md:1]: Remove note from Turquoise killifish. This applies to zebrafish entries only. Tissue state should be changed to NA.

**Review by radtkea — COMMENTED** [review]
Thanks so much for adding these new species! The note referring to transgenic GFP zebrafish lines should be removed. Tissue state should be NA. Thank you!

**Comment by zivy** [comment]
@radtkea I believe I have addressed all the issues. Please take a look and if all is well approve the PR or point out the issues. Thanks.

**Review by radtkea — APPROVED** [review]
Thank you for updating the files. All looks good to me!

**Pull request merged**